### PR TITLE
fix: search input line break

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [ONYX-931] 

### Description

This PR fixes an issue where the search input placeholder breaks on Android.
The issue comes from React-native and is still not fixed even in the latest version.

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

- search input placeholder cropped - mounir

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-931]: https://artsyproduct.atlassian.net/browse/ONYX-931?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Search/Search.tsx [hunk 1]
```diff
@@ -27,7 +27,12 @@ import { SEARCH_PILLS, SEARCH_THROTTLE_INTERVAL, TOP_PILL } from "./constants"
 import { getContextModuleByPillName } from "./helpers"
 import { PillType } from "./types"
 
-const SEARCH_INPUT_PLACEHOLDER = "Search artists, artworks, galleries, etc"
+// This is a workaround for the placeholder text not truncated on android
+// See https://github.com/facebook/react-native/issues/29663
+const SEARCH_INPUT_PLACEHOLDER =
+  Platform.OS === "android" && Dimensions.get("screen").width - 150 < 250
+    ? "Search artists, artworks,  etc"
```
Comment (MounirDhahri) [inline src/app/Scenes/Search/Search.tsx:34]: Here I am simply sending a short version of the placeholder for small devices

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Android user-facing changes (search input placeholder cropped - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b201507834b407c66906b0c8a8ce8737fa343ba1
</p>

File: src/app/Scenes/Search/Search.tsx (hunk 1 above)
Comment (MrSltun) [reply src/app/Scenes/Search/Search.tsx:34]: **Suggestion (non-blocking):** Although this fixes the issue that was reported, I believe that implementing something similar to the workaround (that was introduced [here](https://github.com/artsy/eigen/pull/4833)) over all inputs would be a better solution

Comment (MounirDhahri) [reply src/app/Scenes/Search/Search.tsx:34]: this was unfortunately not used anywhere after this [PR](https://github.com/artsy/eigen/issues/9003) so I removed it from palette. Let me check again if I should bring that back

**Comment by MounirDhahri** [comment]
Here we go https://github.com/artsy/palette-mobile/pull/225

**Pull request closed without merging**